build/Build.md

```diff
@@ -0,0 +1,190 @@
+        *wso2am:3.1.0.1591230650206.3 - 2nd oldest* <br/>
+	    *wso2am:3.1.0.1592240780334.4 - latest* <br/>
+
+        Unique tag: <product>:<version>.<wso2-update-timestamp>.<docker-tag-last-digit>
+
+This queue is regularly updated with a new image having an update (or in other words, a patch) either to the product or docker artifacts. When a new image is pushed, the oldest of the 3 is removed from the registry on a first-in-first-out (FIFO) basis. 
+
+Unique tags are ideal in referring to an image during deployments and we recommend customers to use this tag when they directly use WSO2 images for their deployments without any customizations.
+
+[2] We also maintain one stable tag per each product version. <br/>
+A stable tag is always pointed to the latest docker image of a particular product version. 
+
+Taking the same example from [1], 
+
+* *E.g.* 	
+        <br/>
+        *wso2am:3.1.0.1591023165861.2 - Oldest* <br/>
+        *wso2am:3.1.0.1591230650206.3 - 2nd oldest* <br/>
+	    *wso2am:3.1.0.1592240780334.4 (unique), wso2am:3.1.0 (Stable) - latest* <br/>
+
+        Stable tag: <product>:<version>
+
+Stable tags are ideal in referring to a base image during docker build. We recommend customers to use this tag when they want to build their custom images on top of the product docker images that we host.
+
+[3] We maintain a latest tag per each product. <br/>
+The latest tag is also a stable tag and is always pointed to the latest docker image of a product repo.
+
+Taking the same example from [2], if 3.1.0 is the latest version of wso2am,
+
+* *E.g.* 	
+        <br/>
+        *wso2am:3.1.0.1591023165861.2 - Oldest* <br/>
+        *wso2am:3.1.0.1591230650206.3 - 2nd oldest* <br/>
+	    *wso2am:3.1.0.1592240780334.4 (unique), wso2am:3.1.0 (Stable), wso2am:latest (stable) - latest*
+
+        Latest tag: <product>:latest
+
+WSO2 recommends a similar tagging mechanism to its customers.
+
+Key takeaways:
+
+[1] Uniquely identifying an image is key. <br/>
+[2] Use unique tags for images used directly in your deployments. <br/>
+[3] A stable tag is recommended only for an image used as a base for your deployed image. 
+
+<br/>
+
+## 1.6. Vulnerability scanning and container security check
+
+If a container image includes a vulnerability, then every container generated using the image will also include that vulnerability. Hence it is really important that you scan the newly built image for any vulnerabilities prior to the promotion of the image into a registry.  
+
+Among many other alternative scanning tools, WSO2 uses and would also like to recommend [Clair](https://coreos.com/clair/docs/latest) for this purpose. Clair is an open source project for the static analysis of vulnerabilities in docker containers. Clair scans each container layer and provides a notification of vulnerabilities that may be a threat, based on the Common Vulnerabilities and Exposures database (CVE) and similar databases from Red Hat, Ubuntu, and Debian.
```

Comment:
@DilanUA since we are using [Trivy](https://github.com/aquasecurity/trivy) as a scanning tool, we may have to notify about it as well.

@RidmiR please confirm.

Reply:
Yes, we already plugged Trivy scanner as our docker image vulnerabilities scanner.

Reply:
Clair not anymore in use?

Reply:
Added.

Reply:
Due to a conflict, will be sending the changes in the next PR.